ready/index.js

```diff
@@ -0,0 +1,21 @@
+
+
+const main = () => {
+    
+    document.getElementById("grow-me").classList.add("big")
+    document.getElementById("shrink-me").classList.remove("big")
+
+    const listItems = document.querySelectorAll("ol li");
+    listItems.forEach(item => console.log(item.textContent));
+
+    const link = document.querySelector(".link")
+    link.href = "https://www.apple.com"
```

Comment:
Try to follow the specs

```suggestion
    link.href = "https://www.example.com"
```

Fun fact, example.com is used a lot in code examples because that website is specifically for using as an example. :)